testutils/test_server.go

```diff
@@ -67,12 +71,17 @@ func NewTestServer(t testing.TB, opts *ChannelOpts) *TestServer {
 		ch: comparableState(ch),
 	}
 
-	return &TestServer{
+	ts := &TestServer{
 		TB:            t,
 		channels:      []*tchannel.Channel{ch},
 		channelStates: states,
 		postFns:       opts.postFns,
 	}
+
+	if opts.IncludeRelay {
```

Comment:
For tests that can be run with a relay, I was thinking we should run them twice. Once without, and once with the relay.

This would stay the same, but if someone calls `WithTestServer` with `IncludeRelay`, we should need to copy the options and run with that set to false once.

What do you think?